pkg/apis/core/v1beta1/defaults_shoot.go

```diff
@@ -282,6 +282,14 @@ func SetDefaults_VerticalPodAutoscaler(obj *VerticalPodAutoscaler) {
 		v := DefaultMemoryHistogramDecayHalfLife
 		obj.MemoryHistogramDecayHalfLife = &v
 	}
+	if obj.MemoryAggregationInterval == nil {
+		v := DefaultMemoryAggregationInterval
+		obj.MemoryAggregationInterval = &v
+	}
+	if obj.MemoryAggregationIntervalCount == nil {
+		v := DefaultMemoryAggregationIntervalCount
+		obj.MemoryAggregationIntervalCount = &v
```

Comment:
Any specific reason to not use this and creating a new variable?
```suggestion
		obj.MemoryAggregationIntervalCount = ptr.To(DefaultMemoryAggregationIntervalCount)
```

Reply:
done. I removed all of the intermediate variables for that section.